app/Resources/SchebTwoFactorBundle/views/Authentication/form.html.twig

```diff
@@ -22,7 +22,9 @@
                 </div>
 
                 {% if displayTrustedOption %}
-                    <div class="checkbox"><label for="_trusted"><input id="_trusted" type="checkbox" name="{{ trustedParameterName }}" /> Trust this computer for 30 days</label></div>
+                    <div class="input-group">
```

Comment:
isn't it a missing `form-group` instead ? `input-group` is meant to group the input and the addon.

Reply:
Yep, you're right, I copypasted the wrong class here.  This is what I get for rushing a fix before I've had my coffee... Fixed!